### PR TITLE
LPS-140949 call remote services

### PR DESCRIPTION
**Pull request by javiergamarra**
🤦

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java [hunk 1]
```diff
@@ -101,6 +126,21 @@ public ObjectEntry deleteObjectEntry(long objectEntryId)
 		return objectEntryLocalService.deleteObjectEntry(objectEntryId);
 	}
 
+	@Override
+	public ObjectEntry deleteObjectEntry(
```
Comment (javierdearcos) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java:130]: Usually we get the objectEntry by external reference code and then call delete. It is better like this but I will also call it deleteObjectEntryByExternalReferenceCode to follow the same convention as the get... methods

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java [hunk 2]
```diff
@@ -50,6 +50,19 @@
 )
 public class ObjectEntryServiceImpl extends ObjectEntryServiceBaseImpl {
 
+	@Override
+	public ObjectEntry addObjectEntry(
+			long userId, long groupId, long objectDefinitionId,
```
Comment (javierdearcos) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java:55]: is userId finally need it?

**Review by javierdearcos — APPROVED** [review]
Just added some comments to double check

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java (hunk 2 above)
Comment (nhpatt) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java:55]: The thing is that the manager exposes the userId everywhere and ObjectDDMStorageAdapter passes it from the Form. I think the current use of the adapter does not allow other uses but it's a general adapter, so if someone calls to adapt existing forms we should use the userId of the creator of the form, not the current user executing the adaptation.

So, to be safe/flexible I'm exposing it.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java (hunk 1 above)
Comment (nhpatt) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryServiceImpl.java]: Changed in the rebase.

**Pull request closed without merging**